addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -109,21 +110,53 @@ def __gather_double_sided(blender_material, mesh_double_sided, export_settings):
 
 
 def __gather_emissive_factor(blender_material, export_settings):
-    emissive_socket = gltf2_blender_get.get_socket_or_texture_slot(blender_material, "Emissive")
+    emissive_socket = gltf2_blender_get.get_socket_or_texture_slot_principled(blender_material, "Emission")
+    if emissive_socket is None:
+        emissive_socket = gltf2_blender_get.get_socket_or_texture_slot(blender_material, "Emissive")
```

Comment:
Would this work without adding a new `get_socket_or_texture_slot_principled` function?

```python
 emissive_socket = gltf2_blender_get.get_socket_or_texture_slot(blender_material, "Emission")
if emissive_socket is None:
  emissive_socket = gltf2_blender_get.get_socket_or_texture_slot(blender_material, "Emissive")
```